source/configuration_manual/protocols/lda.rst

```diff
@@ -4,8 +4,7 @@
 Dovecot LDA
 ===========
 
-The Dovecot LDA is a `local delivery
-agent <https://wiki.dovecot.org/MDA>`_, which takes mail from an :ref:`mta`
+The Dovecot LDA is a :ref:`local delivery agent <mda>`, which takes mail from an :ref:`mta`
```

Comment:
This says "[...] is a local delivery agent [...]" and links to the Mail Delivery Agent page.
I think this should say `:ref:mail delivery agent <mda>`